tests/unit/s2n_stream_cipher_null_test.c

```diff
@@ -56,5 +56,26 @@ int main(int argc, char **argv)
         EXPECT_BYTEARRAY_EQUAL(in_array, out_array, out.size);
     };
 
+    /* Test that null cipher is always available */
+    {
+        EXPECT_TRUE(s2n_stream_cipher_null_available());
+    };
+
+    /* Test that get_key always returns success */
+    {
+        struct s2n_blob in = { 0 };
+        EXPECT_OK(s2n_stream_cipher_null_get_key(NULL, &in));
+    };
+
+    /* Test that destroy_key always returns success */
+    {
+        EXPECT_OK(s2n_stream_cipher_null_destroy_key(NULL));
+    };
+
+    /* Test that init always returns success */
+    {
+        EXPECT_OK(s2n_stream_cipher_null_init(NULL));
+    };
```

Comment:
Passing "NULL" in is probably a mistake. These methods are all no-ops so you can pass NULL without a failure, but if they had meaningful implementations then NULL would not be a reasonable input to test "always returns success".

Reply:
Thank you for reviewing.
I defined session_key by referring to the following code. https://github.com/aws/s2n-tls/pull/5268/commits/338684d2c504b294497b7032c63da1d83dbe5d3e
https://github.com/aws/s2n-tls/blob/f013a251f5e2d14231379e1e410e35bced04e887/tests/unit/s2n_tls13_record_aead_test.c#L131